src/messaging/ExchangeContext.cpp

```diff
@@ -670,5 +670,12 @@ void ExchangeContext::ExchangeSessionHolder::GrabExpiredSession(const SessionHan
     GrabUnchecked(session);
 }
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
+void ExchangeContext::OnSessionConnectionClosed(CHIP_ERROR conErr)
+{
+    // TODO: Handle connection closure at the ExchangeContext level.
```

Comment:
Need to create a ticket to track the pending task

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/32250.